trade-finance-manager-ui/server/controllers/case/index.js

```diff
@@ -373,7 +373,7 @@ const getCaseFacility = async (req, res) => {
     allAmendments,
     amendments,
     amendmentsInProgress,
-    showFacilityEndDate: isTfmFacilityEndDateFeatureFlagEnabled() && facility.facilitySnapshot.isGef,
+    showFacilityEndDate: facility.facilitySnapshot.isGef,
```

Comment:
is there a test to update for this change?

Reply:
The tests were siloed into FF on and FF off tests - I've removed the FF off tests & made FF on the default to account for this